tools/walletextension/wallet_extension.go

```diff
@@ -93,17 +90,24 @@ func NewWalletExtension(config Config) *WalletExtension {
 // Serve listens for and serves Ethereum JSON-RPC requests and viewing-key generation requests.
 func (we *WalletExtension) Serve(hostAndPort string) {
 	serveMux := http.NewServeMux()
+
 	// Handles Ethereum JSON-RPC requests received over HTTP.
 	serveMux.HandleFunc(pathRoot, we.handleHTTPEthJSON)
 	serveMux.HandleFunc(PathReady, we.handleReady)
-	// Handles the management of viewing keys.
-	serveMux.Handle(pathViewingKeys, http.StripPrefix(pathViewingKeys, http.FileServer(http.Dir(path.Join(basepath, staticDir)))))
 	serveMux.HandleFunc(PathGenerateViewingKey, we.handleGenerateViewingKey)
 	serveMux.HandleFunc(PathSubmitViewingKey, we.handleSubmitViewingKey)
+
+	// Handles the management of viewing keys.
```

Comment:
Maybe could mention again that it is web assets getting served here, confused me a bit at first. Like `// Static dir contains the web interface for management of viewing keys` or something.